StardewArchipelago/Stardew/StardewItemManager.cs

```diff
@@ -764,5 +763,42 @@ public void ExportAllItemsMatching(System.Func<Object, bool> condition, string f
                 yield return name;
             }
         }
+
+        public void ExportAllMismatchedItems(System.Func<Object, bool> condition, string filePath)

```

Comment:
You should hide this whole block (the two methods) into a #IF DEBUG block, so that it's only accessible from debug mode, like a few other dev tools I made left and right.

Although right now it's not called anywhere, I would also suggest a smapi command in ModEntry to easily call this, also in the debug only list